repo/meta/index.json

```diff
@@ -18,6 +18,23 @@
       }
     },
     {
+      "currentVersion":"0.1.0",
+      "description":"Calico networking for Mesos.  Calico provides IP-per-task functionality with rich policy.  This framework optionally installs Docker multi-host networking (required for Calico with the Docker containerizer) and Mesos Agent network hooks with netmodules (required for Calico with Mesos tasks).  Both installations will cause temporary restarts of an Agent causing transient task loss.  The number of agents restarted at the same time is configurable and defaults to a single concurrent restart.  At the moment, only a limited set of Mesos versions and OSes are supported - check the documentation for details.  The framework will still run on unsupported versions, in which case the Mesos network hooks will not be installed and Calico networking will only be available with the Docker containerizer.",
```

Comment:
would rather state that this is for dcos (instead of mesos).    the underlying details we are abstracting a bit.   Also there is a preference to use the word service over framework.